openml_OS/models/api/v1/Api_run.php

```diff
@@ -510,15 +548,15 @@ private function run_upload() {
       $this->Run->outputData($runId, $did, 'runfile', $key);
     }
 
-    // attach input data
+    // attach input data TODO: JvR: this is legacy and implicit with the task. Remove?
```

Comment:
No longer used? Remove?

Reply:
legacy != no longer used. 

We should revise the db schema to remove this.